features/aanschrijfwijze.feature

```diff
@@ -589,9 +589,9 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
         - er is geen aanspreekvorm als:
           - de persoon geen adellijke titel/predikaat heeft
           - de persoon gebruikt aanduiding naamgebruik "P" (naam partner)
-          - er is geen adellijke titel/predikaat vorm die hoort bij de geslachtsaanduiding van de persoon
-          - de persoon heeft een predikaat, heeft geslachtsaanduiding "V" (vrouw) en heeft een actuele (niet ontbonden) partner
-          - de persoon heeft een predikaat, heeft geslachtsaanduiding "V" (vrouw), heeft geen actuele (maar mogelijk wel ontbonden) partner en heeft aanduiding naamgebruik ongelijk aan "E" (eigen naam) 
+          - er geen adellijke titel/predikaat vorm is die hoort bij de geslachtsaanduiding van de persoon
+          - de persoon een predikaat heeft, de geslachtsaanduiding "V" (vrouw) en een actuele (niet ontbonden) partner
+          - de persoon een predikaat heeft, de geslachtsaanduiding "V" (vrouw), geen actuele (maar mogelijk wel ontbonden) partner en een aanduiding naamgebruik ongelijk aan "E" (eigen naam) 
```

Comment:
waarom verwijder je heeft en voeg je de en een toe?

Reply:
Om een zin te maken die correct is. In feite staat er nu:

_er is geen aanspreekvorm als de persoon heeft een predikaat, heeft geslachtsaanduiding "V" (vrouw), heeft geen actuele (maar mogelijk wel ontbonden) partner en heeft aanduiding naamgebruik ongelijk aan "E" (eigen naam)_

en dan is volgens mij 

_er is geen aanspreekvorm als de persoon een predikaat, de geslachtsaanduiding "V" (vrouw), geen actuele (maar mogelijk wel ontbonden) partner en een aanduiding naamgebruik ongelijk aan "E" (eigen naam) heeft_

beter.